src/app/utils/performance-analysis-logging.spec.ts

```diff
@@ -30,7 +30,7 @@ class TestClass {
   }
 
   @PerformanceAnalysisLogging
-  other(obj?: any) {}
+  other(_obj?: any) {}
```

Comment:
Any special reason for this and similar changes before?

Reply:
Sonarcloud suggested this. The choice was to remove the argument completely or add an underscore before the name. I chose the latter because I think that the semantics matter in this case.

Reply:
I see now, it's the problem with the unused variables